docs/ref/csrf.txt

```diff
@@ -84,29 +84,7 @@ You can acquire the token like this:
 
 .. code-block:: javascript
 
-    function getCookie(name) {
-        let cookieValue = null;
-        if (document.cookie && document.cookie !== '') {
-            const cookies = document.cookie.split(';');
-            for (let i = 0; i < cookies.length; i++) {
-                const cookie = cookies[i].trim();
-                // Does this cookie string begin with the name we want?
-                if (cookie.substring(0, name.length + 1) === (name + '=')) {
-                    cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
-                    break;
-                }
-            }
-        }
-        return cookieValue;
-    }
-    const csrftoken = getCookie('csrftoken');
-
-The above code could be simplified by using the `JavaScript Cookie library
-<https://github.com/js-cookie/js-cookie/>`_ to replace ``getCookie``:
-
-.. code-block:: javascript
-
-    const csrftoken = Cookies.get('csrftoken');
+    let csrftoken = new URLSearchParams(document.cookie).get('csrftoken');
```

Comment:
This needs to be adjusted slightly:

```suggestion
    const csrftoken = new URLSearchParams(document.cookie.replaceAll('; ', '&').get('csrftoken');
```

See, for example, https://gist.github.com/sergeysova/a25a6f1ea6ad358d9d3dd7176e68b6ac.